safe/impact_function_v4/impact_function.py

```diff
@@ -659,6 +566,25 @@ def validate(self):
                     tr('Viewport CRS must be null when an aggregation is '
                        'provided.'))
 
+    def debug_layer(self, layer, name, check_fields=True):
+        """Function executed after each new algorithm if debug mode is on.
```

Comment:
Write the layer produced by each algorithm to the datastore if debug mode is on.